analysis/README.md

```diff
@@ -1,9 +1,35 @@
 # Edna analysis
-
 This folder contains implementation of Edna data analysis in Python.
-
 ## Usage
-
-First of all, normally you don't want to call this code directly, it is supposed to be called from `edna-server`.
-We use [`poetry`](https://python-poetry.org/) for dependency management, though you can install dependencies using your favorite way and not use `poetry`.
+First, normally you don't want to call this code directly, it is supposed to be called from `edna-server`.
+We use [`poetry`](https://python-poetry.org/) for dependency management, though you can install dependencies using your
+favorite way and not use `poetry`.
 To get shell with all dependencies installed run `poetry install` and then `poetry shell` from this directory.
+## Features
+The list of features implemented in an analytic module:
+1. Compute parameters of fitted sigmoid.
+2. Detect potential outliers.
+3. Compute parameters of fitted sigmoid without outliers.
+### Input format
+JSON containing list of experiments in the form:
+```
+[{"experiment": int,
+"data": [[float, float], ...]},
+...]
+```
```

Comment:
Oh, you might have missed this part:
> `findOutliers` is a boolean flag that specifies whether to find outliers.

So if `"findOutliers": false" is passed, you shouldn't compute `outliers` and `new_params`. Actually it won't be a problem if you compute them, I just won't read them. But if it's easy to support this flag, let's support it to do less computations.

Reply:
That's true, will add it.